### PR TITLE
Mark default options correctly in `vue/define-macros-order` docs

### PR DESCRIPTION
**Pull request by wenfangdu**

### CODE REVIEW

**Comment by wenfangdu** [comment]
@ota-meshi Please unblock me, otherwise, I can't receive your messages.

File: docs/rules/define-macros-order.md [hunk 1]
```diff
@@ -27,85 +27,85 @@ This rule reports the `defineProps` and `defineEmits` compiler macros when they
 
 - `order` (`string[]`) ... The order of defineEmits and defineProps macros. You can also add `"defineOptions"` and `"defineSlots"`.
 
-### `{ "order": ["defineProps", "defineEmits"] }` (default)
+### `{ "order": ["defineOptions", "defineProps", "defineEmits", "defineSlots"] }` (default)
```
Comment (FloEdelmann) [inline docs/rules/define-macros-order.md:30]: Why is this order the default? Can you link to examples in the official Vue docs that use this order? Or some explanation?

Comment (FloEdelmann) [inline docs/rules/define-macros-order.md:30]: Also, can you please add or adjust test cases for the new default order?

Comment (wenfangdu) [reply docs/rules/define-macros-order.md:30]: @FloEdelmann Currently, the `eslint-plugin-vue` doc has two defaults, which seems to be done by mistake:
image

Comment (FloEdelmann) [reply docs/rules/define-macros-order.md:30]: It is indeed a mistake in the docs that the second example is marked as "default". That should be fixed in any case, ideally in another PR to fix it without changing the default. Would you fancy doing that?

However, changing the default should only be done if it fits most use cases and/or is the official recommendation from the Vue docs. So can you link to any of those?

Comment (wenfangdu) [reply docs/rules/define-macros-order.md]: Ah, I see. I thought the second one is meant to be the new default, so I adjusted the default order to reflect that, I've pushed a new commit to remove the false default order, please take a look.

**Review by FloEdelmann — APPROVED** [review]
Thanks!

**Pull request merged**